### PR TITLE
Nerves v0.5.0 & Nerves.Bootstrap v0.3.0

### PR DESCRIPTION
**Pull request by mobileoverlord**
Lets do this!

Action packed release

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -6,8 +6,8 @@ defmodule Nerves.Mixfile do
      name: "Nerves",
      source_url: "https://github.com/nerves-project/nerves",
      homepage_url: "http://nerves-project.org/",
-     version: "0.5.0-dev",
-     archives: [nerves_bootstrap: "~> 0.3 or 0.3.0-dev"],
+     version: "0.5.0",
+     archives: [nerves_bootstrap: "~> 0.3.0"],
      elixir: "~> 1.2.4 or ~> 1.3.2 or ~> 1.4",
```
Comment (GregMefford) [inline mix.exs]: I think this should be just `"~>1.4.0"` now.

File: bootstrap/mix.exs [hunk 2]
```diff
@@ -3,7 +3,7 @@ defmodule Nerves.Bootstrap.Mixfile do
 
   def project do
     [app: :nerves_bootstrap,
-     version: "0.3.0-dev",
+     version: "0.3.0",
      elixir: "~> 1.4",
```
Comment (GregMefford) [inline bootstrap/mix.exs]: We should probably make this `"~> 1.4.0"`

File: mix.exs (hunk 1 above)
Comment (mobileoverlord) [inline mix.exs]: Yup you're right

File: bootstrap/mix.exs (hunk 2 above)
Comment (mobileoverlord) [inline bootstrap/mix.exs]: okay, but if all this lock down gets weird, I am totally going to refer back to this conv 😺

**Pull request merged**